source/_integrations/tado.markdown

```diff
@@ -31,6 +31,8 @@ There is currently support for the following device types within Home Assistant:
 
 To use your Tado thermostats in your installation, go to **Configuration** >> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **Tado**.
 
+Previously it was necessary to add your Tado username and password to your `configuration.yaml` file, this is no longer necessary and should be removed.
```

Comment:
This should not be documented. We document the current state of things; this specific log belong in the release notes.